src/bokeh/server/tornado.py

```diff
@@ -280,7 +280,9 @@ def __init__(self,
                  session_token_expiration: int = DEFAULT_SESSION_TOKEN_EXPIRATION,
                  **kwargs: Any):
 
-        from bokeh.application.handlers.document_lifecycle import DocumentLifecycleHandler
+        from bokeh.application.handlers.document_lifecycle import (
+            DocumentLifecycleHandler,
+        )
```

Comment:
This is a bit annoying, though. I'd probably be OK with 100 rather than 88 as the line length. T

Reply:
Maybe this could use relative imports? Then it would fit under 88 character limit.
```py
from ..application.handlers.document_lifecycle import DocumentLifecycleHandler
from ..application.handlers.function import FunctionHandler
```

Reply:
It could, a long time ago the server code used absolute imports since it was not clear if we'd ever split it out to a separate project, but that seems unlikely now.